.lighthouserc-accessibility.js

```diff
@@ -209,6 +210,12 @@ module.exports = {
             'categories:accessibility': ['error', {'minScore': 1}]
           }
         },
+        {
+          'matchingUrlPattern': '^http://127.0.0.1:8181/volunteer$',
+          'assertions': {
+            'categories:accessibility': ['error', {'minScore': 0.91}]
```

Comment:
Why not minScore 1?

Reply:
I ran the lighthouse test locally and the accessibility score came out to 0.91.

Reply:
Done.

Reply:
We no longer use TODOs with usernames (there are some in the codebase but these were created long ago), please add an issue for this.

Reply:
That's what I'd done initially, but the linter failed saying that the TODO format should be `TODO(username): ...`.

Reply:
Wait, what linter? This is probably a mistake.

Reply:
This is irrelevant now since this file has been deleted.

Reply:
No, it wasn't deleted, it was moved/split, please add the URL into .lighthouserc-base.js (you can pick any shard), if you need to modify the min score do so in appropriate .lighthouserc-accessibility-x.js file. Please let me know if you need to know more.

Reply:
I see. I've added it in a shard now, PTAL!